### DISCUSSION TITLE
Disable content validation

### DISCUSSION THREAD

**Discussion by bamod33** [awcodes/filament-tiptap-editor#518] [Ideas]
Hi, I think there is no option to disable html validation, for example if I want to add some HTML like:

```
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    
    etc etc.
```

The editor removes that code

**Comment by awcodes** [comment]
Prosemirror if very strict about what html it allows. That's not really going to be doable without some serious customization of tiptap's extensions.
